src/backend.f90

```diff
@@ -95,7 +96,7 @@ end subroutine reorder
    end interface
 
    abstract interface
-      subroutine sum_yzintox(self, u, u_y, u_z)
+      subroutine sum_intox(self, u, u_)
```

Comment:
I presume this is the start of the refactor mentioned in #38 ?

Reply:
Yes, once we decide how to do it I'll implement in a separate PR. I just had to separate y and z here so that memory reduction can be implemented.